config/pdk-default.php

```diff
@@ -76,4 +77,14 @@
      * Allowed positions for the delivery options in the checkout.
      */
     'deliveryOptionsPositions' => value([]),
+
+    /**
+     * Preferred sort order when displaying carrier names.
+     */
+    'sortOrderCarriers' => value([
```

Comment:
die cijfers zijn niet nodig, de volgorde van de array is de volgorde. Plus, die volgorde kan beter bepaald worden per platform (zie `allowedCarriers` in platform configs). Dan heb je ook geen extra entry nodig en kan je die data gewoon laten leiden